astropy/modeling/functional_models.py

```diff
@@ -33,6 +35,17 @@ class BaseGaussian1D(Fittable1DModel):
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
 
+    def __init__(self, amplitude, mean, stddev, **kwargs):
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        if 'bounds' not in kwargs:
+            kwargs['bounds'] = {'stddev': (FLOAT_EPSILON, None)}
```

Comment:
Just want to caution that in general depending on the model and how the parameter is used this may lead to overflow errors. In #5656 we ended up making it clear in the documentation that the parameter should be positive.